portal-impl/src/com/liferay/portlet/documentlibrary/service/http/DLAppServiceHttp.java

```diff
@@ -332,6 +332,49 @@ public static com.liferay.portal.kernel.repository.model.Folder addFolder(
 		}
 	}
 
+	public static com.liferay.portal.kernel.repository.model.Folder addFolder(
```

Comment:
Is this generated code? In that case we should put it in a commit with generated code